src/main/resources/util/hetero-list-readonly.jelly

```diff
@@ -57,23 +57,23 @@ THE SOFTWARE.
 
     <d:taglib uri="local">
         <d:tag name="body">
-            <table style="width:100%">
+            <div style="width:100%">
```

Comment:
This file should probably be rebased against Jenkins core, forks of core widgets are a pain...

Reply:
Do you mean there is a master copy of this file somewhere to "draw inspiration" from? :)

Reply:
yes here: https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/lib/form/hetero-list.jelly

Reply:
Hi again, I took a look at this and have zero idea what the changes are about :-) 

I updated the file in this repo to `diff` with less noise against the one in upstream (modulo the change from 2-space indents to 4-space, and a few readability blank lines added in this one). But I think it would be beneficial if someone with half an idea about what that markup means does the actual transplants :)

Reply:
This extension point (EnvInjectJobPropertyContributor) is used in 2 plugins: shared objects (up for adoption, known vulnerabilities, 1000 installs) and ontrack (40 installs). Not sure maintaining that extension point worth any time spent